### ISSUE TITLE
Nvidia filters doesn't work with plugin loader

### ISSUE THREAD

**Issue by FantazerYT** [sepluginloader/PluginLoader#57]
"Game is not supported", is there a way to get around that thing or there's no way cuz it's a completely new game for gef experience

**Comment by Souflaki** [comment]
Yeah, i found a way in another Threat. You can copie the extracted SpaceEngineersLauncher file and rename it to anselintegrationtestapp Start it and Tadaaaa
